Countries/SceneDelegate.swift

```diff
@@ -8,45 +8,63 @@
 import UIKit
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
-
+    
     var window: UIWindow?
-
-
+    
+    
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
-        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
-        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        
+        let masterController = ListViewController()
+        let masterNavigationController = UINavigationController(rootViewController: masterController)
+        
+        let detailController = DetailsViewController()
+        let detailNavigationController = UINavigationController(rootViewController: detailController)
+        
+        let splitViewController =  UISplitViewController()
+        splitViewController.viewControllers = [masterNavigationController, detailNavigationController]
+        
+        splitViewController.delegate = self
+        
+        self.window = UIWindow(windowScene: windowScene)
+        self.window?.rootViewController = splitViewController
+        self.window?.makeKeyAndVisible()
+        
+        // scene bounds doesn't reload
     }
-
+    
     func sceneDidDisconnect(_ scene: UIScene) {
-        // Called as the scene is being released by the system.
-        // This occurs shortly after the scene enters the background, or when its session is discarded.
-        // Release any resources associated with this scene that can be re-created the next time the scene connects.
-        // The scene may re-connect later, as its session was not necessarily discarded (see `application:didDiscardSceneSessions` instead).
+        
     }
```

Comment:
Do we need this empty method?

Reply:
Not now.. I deleted them and will add back when/if I need :)